components/remote_retrieval/remote_retrieval.app.mjs

```diff
@@ -1,11 +1,133 @@
+import { axios } from "@pipedream/platform";
+import utils from "./common/utils.mjs";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "remote_retrieval",
-  propDefinitions: {},
+  propDefinitions: {
+    oid: {
+      type: "string",
+      label: "Order ID",
+      description: "The ID of the order to retrieve.",
+      async options({ prevContext }) {
+        const { page } = prevContext;
+        if (page === null) {
+          return [];
+        }
+
+        const {
+          next,
+          results: orders,
+        } = await this.pendingOrders({
+          params: {
+            page,
+          },
+        });
```

Comment:
Are you sure that the response contains `next` and `results`? I haven't tested it, but the [documentation](https://www.remoteretrieval.com/api-documentation/#pending-orders) shows that this endpoint just returns an array.